packages/react-graphql/package.json

```diff
@@ -39,6 +38,7 @@
     "dist/*"
   ],
   "peerDependencies": {
-    "react": "^16.6.0"
+    "react": ">=16.8.0 <17.0.0",
+    "react-apollo": ">=2.2.3 <3.0.0"
```

Comment:
move this to `peerDependencies`, in both case of using `<Query />` and `useMutation`, you really need to had setup `ApolloProvider` for it to work.